record/record_test.go

```diff
@@ -342,7 +349,16 @@ func makeTestRecords(recordLengths ...int) (*testRecords, error) {
 		if err != nil {
 			return nil, err
 		}
-		if _, err = wRec.Write(rec); err != nil {
+		// Don't write records in single calls to Write. Write in multiple cSize chunks.
+		const cSize = 8
+		for j := 0; j < len(rec); j += cSize {
```

Comment:
Instead of the complicated slice expression, you could possibly write it like this:

```
for ; len(rec) > cSize; rec = rec[cSize:] {
  if _, err = wRec.Write(rec[:cSize]); err != nil {
    return nil, err
  }
}
if _, err = wRec.Write(rec); err != nil {
  return nil, err
}
```

and delete the intMin function above.

WDYT?